### PR TITLE
disable about and settings by default

### PR DESCRIPTION
**Pull request by DenysVuika**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [x] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

**What is the new behaviour?**

- disable About and Settings by default

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Review by eromano — APPROVED** [review]

File: src/assets/app.extensions.json [hunk 1]
```diff
@@ -8,6 +8,7 @@
   "$runtime": "1.7.0",
   "$description": "Core application extensions and features",
   "$references": [],
+  "$ignoreReferenceList": ["about.plugin.json", "settings.plugin.json"],
```
Comment (mteodori) [inline src/assets/app.extensions.json:11]: can we use the APP_EXTENSIONS_IGNORE_REFS variable directly here so we don't need the extra shell script? and add a replacer in the startup for the default value

File: docker/default.conf.template [hunk 2]
```diff
@@ -23,6 +23,10 @@ server {
         alias  ${NGINX_ENVSUBST_OUTPUT_DIR}/app.config.json;
     }
 
+    location ~ /assets/app.extensions.json {
```
Comment (mteodori) [inline docker/default.conf.template:26]: did you test with the run.sh script the mapping is as you expect?

File: src/assets/app.extensions.json (hunk 1 above)
Comment (DenysVuika) [reply src/assets/app.extensions.json:11]: not everything runs via docker, using variable there will break the whole development process

File: docker/default.conf.template (hunk 2 above)
Comment (DenysVuika) [reply docker/default.conf.template:26]: still having issues with running the scripts, it seems they need some fixes to be used dev environemnts

Comment (DenysVuika) [reply docker/default.conf.template:22]: no, the `run.sh` and `run-content-ce.sh` not working for me

File: src/assets/app.extensions.json (hunk 1 above)
Comment (mteodori) [reply src/assets/app.extensions.json:11]: the replacer like you have in the other projects supports plain javascript runs maybe @popovicsandras can explain better

**Review by mauriziovitale — APPROVED** [review]

**Pull request merged**